db.py

```diff
@@ -168,17 +168,23 @@ class CourseOrganization(db.Model):
 class Evaluation(db.Model):
     __tablename__ = 'evaluation'
 
-    course_id = db.Column(db.Integer, primary_key=True)
-    course_year = db.Column(db.Integer, primary_key=True)
-    user_id = db.Column(db.Integer, db.ForeignKey('user.id'), primary_key=True)
+    id = db.Column(db.Integer, primary_key=True)
+    course_id = db.Column(db.Integer, nullable=False)
+    course_year = db.Column(db.Integer, nullable=False)
+    user_id = db.Column(db.Integer, db.ForeignKey('user.id'), nullable=False)
     task = db.Column(db.JSON, nullable=False)
+    other_task = db.Column(db.String(100))
     nbr_hours = db.Column(db.String(10), nullable=False)
```

Comment:
You may be able to directly specify that this is an enum.

Reply:
other_task' is not an enum, it corresponds to a textual value.